static/js/components/SubmissionFacets.js

```diff
@@ -71,7 +72,7 @@ export function Option({ option, facetKey }: OptionProps) {
 
   return (
     <div className={className} onClick={cb}>
-      {facetOptionLabels[facetKey](option[labelKey])}
+      {facetOptionLabels[facetKey](labelKey.map(label => option[label]))}
```

Comment:
There's a lot going on in this line. Might be more readable to assign the `labelKey.map` result to a variable